### DISCUSSION TITLE
三月更新 - 2

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#802] [通知]
## 🐛 Bug 修复

- 修复电影本体和PV不能正常切换的问题 @Richasy (#801)
- 修复弹幕速度跳变的问题 @Richasy (#800)
- 修复帮助页面链接滚动布局错位的问题 @Richasy (#799)
- 修复评论详情显示为其它内容的问题 @Richasy (#798)
- 修复下载类型完整但仍标识为仅限视频的问题 @Richasy (#797)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2203.2.0'>三月更新 - 2</a>.</em>

**Comment by Luo-Xiao-Mi** [comment]
大大以后还会更新吗

**Comment by Alpha-lab1** [comment]
有个BUG，默认收藏夹只会显示第一页的收藏内容，无法显示全部的收藏内容

**Comment by DDDandWWW** [comment]
能添加个鼠标中键或侧键返回上一级吗，这真的很好用

**Reply by nlxxtw** [reply]
云大大我爱你，云大大的发布会https://www.bilibili.com/video/BV1JL4y1875w?spm_id_from=333.999.0.0
让我有一种自己有了大会员的错觉，要是能完善就完美了，哈哈
![image](https://user-images.githubusercontent.com/49474003/158588373-afb7eced-c822-46eb-8968-33dba1c821cd.png)
